internal/driver/controllerserver.go

```diff
@@ -119,7 +118,11 @@ func NewControllerServer(ctx context.Context, driver *LinodeDriver, client linod
 // CreateVolume will be called by the CO to provision a new volume on behalf of a user (to be consumed
 // as either a block device or a mounted filesystem).  This operation is idempotent.
 func (cs *ControllerServer) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest) (*csi.CreateVolumeResponse, error) {
+	log, ctx, done := logger.GetLogger(ctx).WithMethod("CreateVolume")
+	defer done()
+
 	name := req.GetName()
+	log.V(2).Info("Processing request", "req", req)
```

Comment:
is there a completed indication for this?

Reply:
Yup if the request was processed completely we have this log:

`log.V(2).Info("Volume created successfully", "response", resp)`